src/pushsource/_impl/model/ami.py

```diff
@@ -99,6 +99,20 @@ class AmiPushItem(VMIPushItem):
     )
     """Billing codes associated with this image."""
 
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Release notes for AMI """
+
+    usage_instructions = attr.ib(type=str, default=None, validator=optional_str)
+    """Instructions for use of AMI"""
+
+    recommended_instance_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Recommended instance type for AMI"""
+
+    product_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Type of product e.g. "AMIProduct", "SaasProduct", "ServerProduct". """
     image_id = attr.ib(type=str, default=None, validator=optional_str)
     """AMI Image ID used to reference the image in AWS."""
```

Comment:
Can you please add a newline to separate these attributes?

Reply:
done.